examples/gte-small-ort/index.ts

```diff
@@ -1,7 +1,9 @@
+const model = new Supabase_UNSTABLE.ai.Session('gte-small');
+
 Deno.serve(async (req: Request) => {
 	const params = new URL(req.url).searchParams;
 	const input = params.get('text');
-	const output = Supabase_UNSTABLE.ai.runModel('gte', input);
+	const output = model.run(input);
```

Comment:
Also wonder if the `run` op should be async...

Reply:
Updated